.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
 -   repo: https://github.com/phantomcyber/dev-cicd-tools
-    rev: v1.17
+    rev: v1.22
```

Comment:
```suggestion
    rev: v1.23
```

Reply:
fair point, on my way to rebuild wheels and rerun tests